### PR TITLE
Defi Oracle Meta Chain Added

### PR DESCRIPTION
**Pull request by Mukulhosen**
---

## PR-Codex overview
This PR updates the version numbers in multiple files to `2.13.4`, adds a new export in `chains/index.ts`, and includes patch notes in `CHANGELOG.md`.

### Detailed summary
- Updated `version` to `2.13.4` in `version.ts`, `jsr.json`, and `package.json`
- Added `defioraclemetaMainnet` export in `chains/index.ts`
- Included patch notes for version `2.13.4` in `CHANGELOG.md`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/chains/index.ts
Comment (jxom) [inline src/chains/index.ts:1]: Please fix this diff.

File: src/chains/definitions/defioraclemeta.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    contracts: {
+        ensRegistry: {
+            address: '0x34eCe27C724291C7233dC8114f76C1169f0863C8',
+        },
+        ensUniversalResolver: {
+            address: '0x231b0Ee14048e9dCcD1d247744d114a4EB5E8E63',
+            blockCreated: 19_971_461,
+        },
+    },
```
Comment (jxom) [inline src/chains/definitions/defioraclemeta.ts:26]: Please remove these. We only accept official ENS contracts for these.

File: src/chains/definitions/defioraclemeta.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { defineChain } from '../../utils/chain/defineChain.js'
+
+export const defioraclemeta = /*#__PURE__*/ defineChain({
+    id: 138,
+    name: 'Defi Oracle Meta Mainnet',
+    nativeCurrency: { name: 'Ether', symbol: 'ETH', decimals: 18 },
+    rpcUrls: {
+        default: {
+            http: ["https://rpc.defi-oracle.io", "wss://wss.defi-oracle.io"],
```
Comment (jxom) [inline src/chains/definitions/defioraclemeta.ts:9]:
```suggestion
            http: ["https://rpc.defi-oracle.io"],
```

File: src/chains/index.ts
Comment (Mukulhosen) [reply src/chains/index.ts:1]: can you help me to change this? I didn't update anything. if start from begin. but the diff still there

**Pull request closed without merging**